docs/usage/working-with-projects.md

```diff
@@ -1,31 +1,41 @@
 # Working with Projects
 
-**Projects** are used to group clusters, to onboard IaaS resources utilized by them and organize access control.
-To work with clusters, you need to create a project that they’ll belong to.
+## Overview
 
-### Prerequisites
-- You have access to the Gardener dashboard and have permissions to create projects.
+Projects are used to group clusters, onboard IaaS resources utilized by them, and organize access control.
+To work with clusters, first you need to create a project that they will belong to.
 
-### Procedure
+## Creating Your First Project
 
-1. Log on to the Gardener Dashboard and choose **CREATE YOUR FIRST PROJECT**.
+### Prerequisites
+- You have access to the Gardener Dashboard and have permissions to create projects
 
-    <img class="gs-image" src="../images/00-create-project.png">
+### Steps
 
-2. Provide a project **Name**, and optionally a **Description**, and a **Purpose**, and choose **CREATE**.
+1. Logon to the Gardener Dashboard and choose **CREATE YOUR FIRST PROJECT**.
+  
+    <img class="gs-image" src="../images/00-create-project.png">
 
-    **Note:** You will **not** be able to change the project **Name** later. The rest of the details are editable.
+2. Provide a project **Name**, and optionally a **Description**, **Cost Center** and a **Purpose**, and choose **CREATE**.
```

Comment:
```suggestion
2. Provide a project **Name**, and optionally a **Description** and a **Purpose**, and choose **CREATE**.
```
also remove the cost center part from the screenshot below

Reply:
In order to provide a screenshot without the Cost Center part, I would need access to a Gardener Dashboard that lacks the Cost Center option in order to take an undoctored screenshot, or such a screenshot provided to me.

Reply:
An option to proceed without a new screenshot would be to simply blur out the Cost Center requirement. Would that work for you?

Reply:
yes  👍